lib/node_modules/@stdlib/stats/base/dists/levy/quantile/benchmark/benchmark.js

```diff
@@ -31,18 +32,26 @@ var quantile = require( './../lib' );
 // MAIN //
 
 bench( pkg, function benchmark( b ) {
+	var len;
 	var mu;
 	var c;
 	var p;
 	var y;
 	var i;
 
+	len = 100;
+	p = new Float64Array( len );
+	mu = new Float64Array( len );
+	c = new Float64Array( len );
+	for ( i = 0; i < len; i++ ) {
+		p[ i ] = randu();
```

Comment:
```suggestion
		p[ i ] = unifrom( 0.0, 1.0 );
```
as the logic in `benchmark.c` should resonate with the js file so for added clarity use `uniform` function to generate the paraemeters through out the js file.